src/main/java/seedu/jarvis/model/masterycheck/MasteryCheckNameContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,30 @@
+package seedu.jarvis.model.masterycheck;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.jarvis.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code MasteryCheck}'s student name matches any of the keywords given.
+ */
+public class MasteryCheckNameContainsKeywordsPredicate implements Predicate<MasteryCheck> {
```

Comment:
Great naming the class. now when any code calls MasteryCheckNameContainsKeywordsPredicate.test(mc) it makes a lot of sense!